src/app/store/test-records/effects/test-records.effects.ts

```diff
@@ -169,11 +170,16 @@ export class TestResultsEffects {
       ofType(contingencyTestTypeSelected),
       mergeMap(action =>
         of(action).pipe(
-          withLatestFrom(this.store.select(testResultInEdit), this.store.select(selectTestType(action.testType)), this.userService.user$),
+          withLatestFrom(
+            this.store.select(testResultInEdit),
+            this.store.select(selectTestType(action.testType)),
+            this.userService.user$,
+            this.store.select(getTestStationFromProperty('testStationType', TestStationType.HQ))
```

Comment:
Why would you have store, store, users, store... Nothing wrong with it, but doesn't it hurt your eyes? 😂

Reply:
Eh... I don't really mind 😆

Reply:
This needs to be fixed